include/CppUTest/TestRegistry.h

```diff
@@ -33,6 +33,7 @@
 #ifndef D_TestRegistry_h
 #define D_TestRegistry_h
 
+#include <stddef.h>
```

Comment:
You'll need to include the StdC include instead of including Standard C directly.